libs/order/driver/magento/2.4.1/src/queries/fragments/order-item.ts

```diff
@@ -18,6 +18,7 @@ export const orderItemFragment = gql`
     product_url_key
     product_sku
     product_name
+    product_type
```

Comment:
Is this not a __typename?

Reply:
I guess it could be implemented with that but no, `product_type` is an actual field returned on the order item model.

Reply:
bump. we need to use `product_type` to differentiate between simple and configurables.